src/content_script.js

```diff
@@ -652,7 +692,7 @@ async function detectFacebookOnPage () {
 
   // Check if user dismissed the Relay prompt
   const relayAddonPromptDismissed = await getLocalStorageSettingFromBackground("hideRelayEmailBadges");
-  if (!relayAddonEnabled && !relayAddonPromptDismissed.hideRelayEmailBadges && trackersDetectedOnCurrentPage) {
+  if (relayAddonPromptDismissed && !relayAddonEnabled && !relayAddonPromptDismissed.hideRelayEmailBadges && trackersDetectedOnCurrentPage) {
```

Comment:
I kept getting console errors using this. 

To check, visit airbnb.com. You should see an email prompt (if you do not have Relay installed and have not opt'ed out of the labels)